snaps/how-to/test-a-snap.md

```diff
@@ -5,18 +5,43 @@ sidebar_position: 6
 
 # Test a Snap
 
-You can test your Snap by hosting it locally using `yarn start`, installing it in Flask, and calling
-its API methods from a dapp.
+You can test your Snap [locally](#test-locally), in the [Snaps sandbox](#test-in-a-sandbox), and [end-to-end using Jest](#test-end-to-end).
 
-For end-to-end Snap testing in a Jest environment, use the
-[`@metamask/snaps-jest`](https://github.com/MetaMask/snaps/tree/main/packages/snaps-jest) package
-as follows.
+## Test locally
 
-## Steps
+1. Host your Snap locally:
+
+   ```bash
+   yarn start
+   ```
+   
+2. Install your Snap in Flask.
+
+3. Test your Snap by calling its API methods from a dapp.
+
+## Test in the sandbox
+
+Use the Snaps sandbox to easily test and debug your Snap.
```

Comment:
```suggestion
Use the Snaps sandbox to test and debug your Snap.
```

Reply:
"easily" is kind of the key benefit of testing via the sandbox vs testing locally. Maybe "in an easy-to-use interface" is more explanatory?